main.py

```diff
@@ -19,13 +19,29 @@
 
 from CanvasAPI import CanvasAPI
 
-import secrets
+# secrets really is used during (import to change sensitive properties).
+import secrets  # @UnusedImport
 import util
 
+##### Improved code tracebacks for exceptions
+import traceback
+
+def handleError(self, record):  # @UnusedVariable
+    traceback.print_stack()
+logging.Handler.handleError = handleError
+#####
+
 TIMEZONE_UTC = dateutil.tz.tzutc()
 RUN_START_TIME = datetime.datetime.now(tz=TIMEZONE_UTC)
 RUN_START_TIME_FORMATTED = RUN_START_TIME.strftime('%Y%m%d%H%M%S')
 
+# Hold parsed options
+options = None
+
+# Level to use for all logging
+#logging_level = logging.DEBUG
+logging_level = logging.INFO
```

Comment:
Rename to `loggingLevel`.

Reply:
I noticed [the other one](https://github.com/tl-its-umich-edu/kartograafr/pull/14#discussion_r152436624) before this one.

Reply:
Ok, for consistency I'll use camelcase.